<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(255,255,255);
}

QPushButton#xButton
{
	background: transparent;
	border-image: url(:/Image/X.png);
}

QPushButton#minButton
{
	background: transparent;
	border-image: url(:/Image/min.png);
}

QPushButton#settingButton
{
	background: transparent;
	border-image: url(:/Image/setting.png);
}

QPushButton#xButton:hover
{
	background-color: rgba(255, 78, 73, 128);
	border-image: url(:/Image/X 2.png);
}

QPushButton#minButton:hover
{
	background-color: rgba(128, 128, 128, 128);
	border-image: url(:/Image/min 2.png);
}

QPushButton#settingButton:hover
{
	background-color: rgba(128, 128, 128, 128);
	border-image: url(:/Image/setting 2.png);
}

QPushButton#keyboardButton
{
	background: transparent;
	border-image: url(:/Image/keyboard 2.png);
}

QPushButton#keyboardButton:hover
{
	border-image: url(:/Image/keyboard.png);
}

QPushButton#QRcodeButton
{
	background: transparent;
	border-image: url(:/Image/QRcode 2.png);
}

QPushButton#QRcodeButton:hover
{
	border-image: url(:/Image/QRcode.png);
}

QPushButton#findButton
{
	background: transparent;
}

QPushButton#findButton:hover
{
	font: 700 14pt &quot;Heiti SC&quot;;
}

QPushButton#signButton
{
	background: transparent;
}

QPushButton#signButton:hover
{
	font: 700 14pt &quot;Heiti SC&quot;;
}

QLabel#accountPix
{
	background: transparent;
	border-image: url(:/Image/account.png);
}

QLabel#pwdPix
{
	background: transparent;
	border-image: url(:/Image/lock.png);
}

QLineEdit#pwdEdit
{
	background: transparent;
	border: none;
	border-bottom: 1px solid rgb(200, 200, 200);
	font: bold 16px;
}

QLineEdit#pwdEdit:hover
{
	border-bottom: 1px solid rgb(255, 255, 255);
}

QLineEdit#pwdEdit:focus
{
	border-bottom: 1px solid rgb(150, 249, 255);
}

QRadioButton#autoButton
{
	background: transparent;
}

QRadioButton#saveButton
{
	background: transparent;
}

QPushButton#loginButton
{
	background-color: rgba(107, 176, 255, 128);
	border-radius: 5px;
	color: white
}

QPushButton#loginButton:hover
{
	background-color: rgba(73, 116, 185, 196);
}

QComboBox#userBox
{
	background: transparent;
	border: none;
	border-bottom: 1px solid rgb(200, 200, 200);
	font: bold 16px;
}

QComboBox#userBox:hover
{
	border-bottom: 1px solid rgb(255, 255, 255);
}

QComboBox#userBox:focus
{
	border-bottom: 1px solid rgb(150, 249, 255);
}

QComboBox#userBox::drop-down
{
	min-width: 25px;
	border-image: url(:/Image/down 2.png);
}

QComboBox#userBox::drop-down:hover
{
	border-image: url(:/Image/down.png);
}

QComboBox#userBox::drop-down:checked
{
	border-image: url(:/Image/up.png);
}

QComboBox QAbstractItemView {
	outline: 0px solid rgb(211, 246, 255);   /* 选定项的虚框 */
	border: 1px solid rgb(150, 249, 255);   /* 整个下拉窗体的边框 */
	border-top: 0px;
	background-color: none;   /* 整个下拉窗体的背景色 */
	selection-background-color: rgb(200, 200, 200);   /* 整个下拉窗体被选中项的背景色 */
}</string>
  </property>
  <widget class="QLabel" name="labTop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="src.qrc">:/Image/1.gif</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="xButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="minButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>0</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="settingButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>0</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="accountPix">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>134</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pwdPix">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="autoButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>自动登录</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="keyboardButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>185</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="QRcodeButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>360</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="findButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="signButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>注册账号</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>310</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
  </widget>
  <widget class="QComboBox" name="userBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>793138445</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>297358412</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
